.github/workflows/ci.yml

```diff
@@ -28,20 +48,22 @@ jobs:
       - name: Restore dependencies cache
         uses: actions/cache@v3
         with:
-          path: server/deps
+          path: deps
           key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
           restore-keys: ${{ runner.os }}-mix-
       - name: Install dependencies
-        run: mix deps.get
+        run: make setup
+      - name: Credo Elixir Check
+        run: make credo
       - name: Check Elixir Formatting
         run: mix format --check-formatted
-      - name: Check Elixir code
-        run: mix credo --strict
-      - name: Check Rust Formatting
-        run: cargo fmt --check --manifest-path native/lambda_game_engine/Cargo.toml
-      - name: Check Rust code
-        run: cargo clippy --manifest-path native/lambda_game_engine/Cargo.toml -- -D warnings
+      - name: (Load Test) Check Elixir Formatting
+        run: cd load_test && mix format --check-formatted
       - name: Compiles without warnings
         run: mix compile --all-warnings --warnings-as-errors
       - name: Run tests
-        run: mix test
+        run: make tests
+      - name: Check Rust Formatting
+        run: cargo fmt --check --manifest-path native/game_backend/Cargo.toml
+      - name: Check Rust code
+        run: cargo clippy --manifest-path native/game_backend/Cargo.toml -- -D warnings
```

Comment:
This checks should probably go before the tests

Reply:
Yeah I agree. Moved this before the tests.